### ISSUE TITLE
Failure in using GAMESS to obtain a correct structure of BeCl2

### ISSUE THREAD

**Issue by Thepcuchem** [markperri/chemcompute_issues#7]
Failure in using GAMESS to obtain a correct structure of BeCl2 after optimization

**Comment by markperri** [comment]
Hi, what is the job number?  I'm looking at recent BeCl2 jobs and they look linear with a bond length of 1.8 A.  For example: 
 https://chemcompute.org/gamess/status/870992

**Comment by Thepcuchem** [comment]
I did not use DFT. I simply type BeCl2 in the "Search" box. After done symmetrization, it shows a weird model on 3D Panel. I think the program used MM method at this stage.
________________________________

**Comment by markperri** [comment]
Yes, we use MM to get an initial 3D structure guess in your browser, and MM predicts bent.  Once you calculate using GAMESS it will optimize to a linear structure.

**Comment by Thepcuchem** [comment]
It is not in my case. Eventhough I turned off the "Add hydrogens automatically" option, the program still added a hydrogen attaching to Cl. The final model became Be ClH ClH without bonding. Please see the attach below.

My intension is to teach students a rapid way in computational chemistry to predict a correct geometry of compounds to compare with the VSEPR theory without going into a deep quantum chemistry. I did not have this problem for other compounds such as HgCl2, BF3, CH4, H2O, XeF4. They were built using the same protocol (via a simple "search" box).

[cid:5edc6d68-30ff-4c60-9187-4032bed09dda]
________________________________

**Comment by markperri** [comment]
Yes, it does add hydrogens in the 2D panel, but it won't have them in the 3d panel.  I think it's because the bond length is so long that it is making HCl out of what it thinks are lone Cl atoms.  The database search gets data from the 3D crystal structure, not gas phase data.  You can explain to them that there is no bond information in the database, and thus it has to guess at bonds based on distance.  This fails with BeCl2, hence all the problems.

The QM calculation only takes 1 minute, so you should have your students do the whole calculation to see a better gas phase structure.

**Comment by Thepcuchem** [comment]
Thank you for your responds. The thing is there will be about 150-200 students submitting the job at the same time. This causes traffic.

Honestly, I appreciate your time for taking action of this.
________________________________

**Comment by markperri** [comment]
I don't think 150 - 200 students will cause a problem, especially with this short of a job.  It's a good example of a lot of common things we face:  lack of bonding information leads to assumptions / guesses.  Failure of classical mechanics to predict 3d structure.  Success of quantum mechanics.  So it sounds like a good exercise.

**Comment by Thepcuchem** [comment]
Thank you for your suggestion. I understand the server is powerful and shouldn't be a big deal. There will be several compounds to be predicted in this exercise. This could cause the internet traffice.
Besides, I prefer a bit consistency for the protocol, QM calculations have several special words such as basis set, 6-31G*, AM1, B3LYP, RHF.  Explaining these jargons will be overwhelming for my students.
At the moment, they are not ready.

Thank you

________________________________
